src/components/20-molecules/dropdown/index.scss

```diff
@@ -40,7 +40,7 @@ axa-dropdown {
   display: block;
 
   .m-dropdown__label {
-    @include text(t3);;
+    @include text(t3);
```

Comment:
nice

Reply:
Yeah, these are the real benefits. It formats your code when it can be "compiled" (makes it more robust during coding), so you get instant feedback if what you are doing is legit. :)